### PR TITLE
data: akashrajakku

### PR DESCRIPTION
**Pull request by akashrajakku**

### CODE REVIEW

File: data/akashraj.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Akash Raj",
+  "type": "personal",
+  "bio": "Open-Source,Web-Dev,DSA",
+  "avatar":"https://github.com/akashrajakku.png",
```
Comment (Panquesito7) [inline data/akashraj.json:5]: The `avatar` field is no longer required as of a recent update.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Your filename must match your GitHub username.
In this case, it should be `akashrajakku`. Please update accordingly.

If you need any help, let us know. 🙂

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SurajPratap10 — APPROVED** [review]
Good

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by akashrajakku** [comment]
Thanks for having me

**Pull request merged**